README.md

```diff
@@ -10,7 +10,7 @@
 
 This project contains tools, scripts, and _best-known-configuration_ (BKC) for
 Linux guest kernel hardening in the context of Confidential Cloud Computing threat
-model.
+model. For motivation and solution overview, refer to [Guest Hardening Strategy](https://intel.github.io/ccc-linux-guest-hardening-docs/tdx-guest-hardening.html#).
```

Comment:
line wrap?

Reply:
Broke the line in the source.
Not sure if I understand this point correctly.  Could you please explain if I did not?